From: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Date: Sat, 22 Oct 2016 20:43:40 +0200
Subject: doc: fix spelling errors

Thanks to Mathieu Malaterre <malat@debian.org> for reporting the
Que/Queue typo. (https://bugs.debian.org/839542)

Reviewed-by: Lou Logan <lou@lrcd.com>
Signed-off-by: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
---
 doc/encoders.texi               | 2 +-
 doc/ffprobe.texi                | 2 +-
 doc/filters.texi                | 4 ++--
 ffmpeg.c                        | 2 +-
 ffmpeg_cuvid.c                  | 4 ++--
 libavcodec/aaccoder_twoloop.h   | 2 +-
 libavcodec/cabac.c              | 2 +-
 libavcodec/ffjni.c              | 2 +-
 libavcodec/mediacodec_wrapper.h | 2 +-
 libavcodec/psymodel.h           | 2 +-
 libavcodec/x86/vp9lpf_16bpp.asm | 2 +-
 libavfilter/f_ebur128.c         | 2 +-
 libavformat/internal.h          | 2 +-
 libavutil/frame.h               | 2 +-
 libavutil/tree.h                | 2 +-
 15 files changed, 17 insertions(+), 17 deletions(-)

diff --git a/doc/encoders.texi b/doc/encoders.texi
index f38cad3..f2a9950 100644
--- a/doc/encoders.texi
+++ b/doc/encoders.texi
@@ -1866,7 +1866,7 @@ Enable CAVLC and disable CABAC. It generates the same effect as
 @end table
 
 @item cmp
-Set full pixel motion estimation comparation algorithm. Possible values:
+Set full pixel motion estimation comparison algorithm. Possible values:
 
 @table @samp
 @item chroma
diff --git a/doc/ffprobe.texi b/doc/ffprobe.texi
index 2024eed..1069ae3 100644
--- a/doc/ffprobe.texi
+++ b/doc/ffprobe.texi
@@ -245,7 +245,7 @@ continue reading from that.
 Each interval is specified by two optional parts, separated by "%".
 
 The first part specifies the interval start position. It is
-interpreted as an abolute position, or as a relative offset from the
+interpreted as an absolute position, or as a relative offset from the
 current position if it is preceded by the "+" character. If this first
 part is not specified, no seeking will be performed when reading this
 interval.
diff --git a/doc/filters.texi b/doc/filters.texi
index b482236..c602c72 100644
--- a/doc/filters.texi
+++ b/doc/filters.texi
@@ -1120,7 +1120,7 @@ Set video stream size. Only useful if curves option is activated.
 
 @item mgain
 Set max gain that will be displayed. Only useful if curves option is activated.
-Setting this to reasonable value allows to display gain which is derived from
+Setting this to a reasonable value makes it possible to display gain which is derived from
 neighbour bands which are too close to each other and thus produce higher gain
 when both are activated.
 
@@ -16753,7 +16753,7 @@ magnitude across time and second represents phase across time.
 The filter will transform from frequency domain as displayed in videos back
 to time domain as presented in audio output.
 
-This filter is primarly created for reversing processed @ref{showspectrum}
+This filter is primarily created for reversing processed @ref{showspectrum}
 filter outputs, but can synthesize sound from other spectrograms too.
 But in such case results are going to be poor if the phase data is not
 available, because in such cases phase data need to be recreated, usually
diff --git a/ffmpeg.c b/ffmpeg.c
index cdded86..80c76dd 100644
--- a/ffmpeg.c
+++ b/ffmpeg.c
@@ -3546,7 +3546,7 @@ static int check_keyboard_interaction(int64_t cur_time)
                         "+      increase verbosity\n"
                         "-      decrease verbosity\n"
                         "c      Send command to first matching filter supporting it\n"
-                        "C      Send/Que command to all matching filters\n"
+                        "C      Send/Queue command to all matching filters\n"
                         "D      cycle through available debug modes\n"
                         "h      dump packets/hex press to cycle through the 3 states\n"
                         "q      quit\n"
diff --git a/ffmpeg_cuvid.c b/ffmpeg_cuvid.c
index 7fb47a2..fe9ce84 100644
--- a/ffmpeg_cuvid.c
+++ b/ffmpeg_cuvid.c
@@ -187,7 +187,7 @@ int cuvid_transcode_init(OutputStream *ost)
     }
 
     /* This is a bit hacky, av_hwframe_ctx_init is called by the cuvid decoder
-     * once it has probed the neccesary format information. But as filters/nvenc
+     * once it has probed the necessary format information. But as filters/nvenc
      * need to know the format/sw_format, set them here so they are happy.
      * This is fine as long as CUVID doesn't add another supported pix_fmt.
      */
@@ -229,7 +229,7 @@ error:
 
 cancel:
     if (ist->hwaccel_id == HWACCEL_CUVID) {
-        av_log(NULL, AV_LOG_ERROR, "CUVID hwaccel requested, but impossible to achive.\n");
+        av_log(NULL, AV_LOG_ERROR, "CUVID hwaccel requested, but impossible to achieve.\n");
         return AVERROR(EINVAL);
     }
 
diff --git a/libavcodec/aaccoder_twoloop.h b/libavcodec/aaccoder_twoloop.h
index 42aea52..e44f69a 100644
--- a/libavcodec/aaccoder_twoloop.h
+++ b/libavcodec/aaccoder_twoloop.h
@@ -87,7 +87,7 @@ static void search_for_quantizers_twoloop(AVCodecContext *avctx,
      * will keep iterating until it fails to lower it or it reaches
      * ulimit * rdlambda. Keeping it low increases quality on difficult
      * signals, but lower it too much, and bits will be taken from weak
-     * signals, creating "holes". A balance is necesary.
+     * signals, creating "holes". A balance is necessary.
      * rdmax and rdmin specify the relative deviation from rdlambda
      * allowed for tonality compensation
      */
diff --git a/libavcodec/cabac.c b/libavcodec/cabac.c
index c0abe83..dd2b057 100644
--- a/libavcodec/cabac.c
+++ b/libavcodec/cabac.c
@@ -182,7 +182,7 @@ int ff_init_cabac_decoder(CABACContext *c, const uint8_t *buf, int buf_size){
 #if CABAC_BITS == 16
     c->low =  (*c->bytestream++)<<18;
     c->low+=  (*c->bytestream++)<<10;
-    // Keep our fetches on a 2-byte boundry as this should avoid ever having to
+    // Keep our fetches on a 2-byte boundary as this should avoid ever having to
     // do unaligned loads if the compiler (or asm) optimises the double byte
     // load into a single instruction
     if(((uintptr_t)c->bytestream & 1) == 0) {
diff --git a/libavcodec/ffjni.c b/libavcodec/ffjni.c
index 82ee5d3..cb1884f 100644
--- a/libavcodec/ffjni.c
+++ b/libavcodec/ffjni.c
@@ -215,7 +215,7 @@ int ff_jni_exception_get_summary(JNIEnv *env, jthrowable exception, char **error
     } else if (!name && message) {
         av_bprintf(&bp, "Exception: %s", message);
     } else {
-        av_log(log_ctx, AV_LOG_WARNING, "Could not retreive exception name and message\n");
+        av_log(log_ctx, AV_LOG_WARNING, "Could not retrieve exception name and message\n");
         av_bprintf(&bp, "Exception occurred");
     }
 
diff --git a/libavcodec/mediacodec_wrapper.h b/libavcodec/mediacodec_wrapper.h
index cddd420..1b4f3a9 100644
--- a/libavcodec/mediacodec_wrapper.h
+++ b/libavcodec/mediacodec_wrapper.h
@@ -44,7 +44,7 @@
  * implementation.
  *
  * The API around MediaCodecList is not part of the NDK (and is lacking as
- * we still need to retreive the codec name to work around faulty decoders
+ * we still need to retrieve the codec name to work around faulty decoders
  * and encoders).
  *
  * For documentation, please refers to NdkMediaCodec.h NdkMediaFormat.h and
diff --git a/libavcodec/psymodel.h b/libavcodec/psymodel.h
index 35d184c..582f040 100644
--- a/libavcodec/psymodel.h
+++ b/libavcodec/psymodel.h
@@ -29,7 +29,7 @@
 /** maximum number of channels */
 #define PSY_MAX_CHANS 20
 
-/* cutoff for VBR is purposedly increased, since LP filtering actually
+/* cutoff for VBR is purposely increased, since LP filtering actually
  * hinders VBR performance rather than the opposite
  */
 #define AAC_CUTOFF_FROM_BITRATE(bit_rate,channels,sample_rate) (bit_rate ? FFMIN3(FFMIN3( \
diff --git a/libavcodec/x86/vp9lpf_16bpp.asm b/libavcodec/x86/vp9lpf_16bpp.asm
index c15437b..c088817 100644
--- a/libavcodec/x86/vp9lpf_16bpp.asm
+++ b/libavcodec/x86/vp9lpf_16bpp.asm
@@ -78,7 +78,7 @@ SECTION .text
 %endif
 %endmacro
 
-; calulate p or q portion of flat8out
+; calculate p or q portion of flat8out
 %macro FLAT8OUT_HALF 0
     psubw               m4, m0                      ; q4-q0
     psubw               m5, m0                      ; q5-q0
diff --git a/libavfilter/f_ebur128.c b/libavfilter/f_ebur128.c
index 59eaedd..983c889 100644
--- a/libavfilter/f_ebur128.c
+++ b/libavfilter/f_ebur128.c
@@ -141,7 +141,7 @@ typedef struct {
     int loglevel;                   ///< log level for frame logging
     int metadata;                   ///< whether or not to inject loudness results in frames
     int dual_mono;                  ///< whether or not to treat single channel input files as dual-mono
-    double pan_law;                 ///< pan law value used to calulate dual-mono measurements
+    double pan_law;                 ///< pan law value used to calculate dual-mono measurements
 } EBUR128Context;
 
 enum {
diff --git a/libavformat/internal.h b/libavformat/internal.h
index 647ad65..31214f9 100644
--- a/libavformat/internal.h
+++ b/libavformat/internal.h
@@ -626,7 +626,7 @@ int ff_bprint_to_codecpar_extradata(AVCodecParameters *par, struct AVBPrint *buf
  * The packet is not removed from the interleaving queue, but only
  * a pointer to it is returned.
  *
- * @param ts_offset the ts difference between packet in the que and the muxer.
+ * @param ts_offset the ts difference between packet in the queue and the muxer.
  *
  * @return a pointer to the next packet, or NULL if no packet is queued
  *         for this stream.
diff --git a/libavutil/frame.h b/libavutil/frame.h
index 2b5c332..ef87806 100644
--- a/libavutil/frame.h
+++ b/libavutil/frame.h
@@ -178,7 +178,7 @@ typedef struct AVFrameSideData {
  * without breaking compatibility with each other.
  *
  * Fields can be accessed through AVOptions, the name string used, matches the
- * C structure field name for fields accessable through AVOptions. The AVClass
+ * C structure field name for fields accessible through AVOptions. The AVClass
  * for AVFrame can be obtained from avcodec_get_frame_class()
  */
 typedef struct AVFrame {
diff --git a/libavutil/tree.h b/libavutil/tree.h
index 9a9e11b..d5e0aeb 100644
--- a/libavutil/tree.h
+++ b/libavutil/tree.h
@@ -58,7 +58,7 @@ struct AVTreeNode *av_tree_node_alloc(void);
  *             then the corresponding entry in next is unchanged.
  * @param cmp compare function used to compare elements in the tree,
  *            API identical to that of Standard C's qsort
- *            It is guranteed that the first and only the first argument to cmp()
+ *            It is guaranteed that the first and only the first argument to cmp()
  *            will be the key parameter to av_tree_find(), thus it could if the
  *            user wants, be a different type (like an opaque context).
  * @return An element with cmp(key, elem) == 0 or NULL if no such element
